src/components/Palette/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import SchemeColor from '../SchemeColor'
+import './../../components/Palette/style.css'
```

Comment:
technicky vzato stačí jen 
```
import '../../components/Palette/style.css'
```
ale je to je taková kosmetická záležitost, funguje stejně